app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource
+
+    def index
+      @conferences_available = Conference.with_roles([:admin, :organizer, :cfp], current_user)
```

Comment:
Hi,
I'm helping out @raluka working on this. We have a hard time coming up with a query that enables us to display the comments grouped by events title. Could you give us a hint on how your implementation would look like? :)